transcom/mymove
# B 19157

[B-19157](https://www13.v1host.com/USTRANSCOM38/Member.mvc/Summary?oidToken=Member%3A914376&feat-nav=-m1)

[B-19157-INT Pull request](https://github.com/transcom/mymove/pull/12284)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Access the customer application
Login as any role
Select "Privacy and Security Policy" at the bottom of the page
Verify the information is updated (screenshot)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-03-18 at 12 26 30 PM](https://github.com/transcom/mymove/assets/162157577/b91b6823-7f93-4e04-9084-d64d6f5c4f57)
